lxd/api_internal.go

```diff
@@ -795,6 +795,14 @@ func internalImportFromBackup(d *Daemon, projectName string, instName string, fo
 	// restore that changes that instance's place in time (moving it backwards)
 	instDBArgs.Config["volatile.uuid"] = uuid.New()
 
+	// If the instance is a VM, generate a new `volatile.uuid.vmgenid`
+	// to differentiate this backed up VM from the original.
+	if instDBArgs.Type == instancetype.VM {
+		instDBArgs.Config["volatile.uuid.vmgenid"] = uuid.New()
```

Comment:
My understanding from https://github.com/lxc/lxd/issues/11449 is that we will just use `volatile.uuid.generation` for both VMs and containers. @stgraber do you concur?